README.md

```diff
@@ -1,146 +1,195 @@
+- Introduction of Single and Multi Account Public Client Applications
+- IAccount and access to claims
+- Support for synchronous methods from worker threads
+- Improved configuration and control of your PublicClientApplication using configuration file
+- Implemented using AndroidX
 
-* supported devices, API levels (need help here)
+### Migrating from ADAL
 
+You can review the ADAL to MSAL migration guide here
 
-## Installation
+### Migrating from preview versions of MSAL
 
-For a full example, checkout the [quickstart](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-v2-android) and [code sample](https://github.com/Azure-Samples/ms-identity-android-java).
+You can review the MSAL Preview to GA release guide here
 
-### Binaries via Gradle (Recommended way)
-** This needs to be updated
+### Sample
+
+For a complete running sample [link to new sample]()
+
+## Using MSAL
+
+- Before you can get a token from Azure AD v2.0 or Azure AD B2C, you'll need to register an application. To register your app, use [the Azure portal](https://aka.ms/AppRegistrationsPreview). For Azure AD B2C, checkout [how to register your app with B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-app-registration).  
+
+### Requirements
+
+- Android API Level 16+
+
+### Step 1: Declare dependency on MSAL
 
 Add to your app's build.gradle:
 
 ```gradle
-    dependencies {
-        implementation 'com.microsoft.identity.client:msal:1.0.+'
-        }
-    }
+dependencies {
+    implementation 'com.microsoft.identity.client:msal:1.0.+'
+}
 ```
 
-### AAR package inside libs folder
-You can get the AAR file from maven central (Link??) and drop it into **libs** folder of your project.
+### Step 2: Create your MSAL configuration file
 
-## Using MSAL
-** I think a lot of this needs to be updated
+>NOTE: Add link to configuration file documentation here
 
-- Make sure you've included MSAL in your app's *build.gradle*.
-- Before you can get a token from Azure AD v2.0 or Azure AD B2C, you'll need to register an application. To register your app, use [the Azure portal](https://aka.ms/AppRegistrationsPreview). For Azure AD B2C, checkout [how to register your app with B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-app-registration).  
+It's simplest to create your configuration file as a "raw" resoruce file in your project resources.  You'll be able to refer to this using the generated resource identifier when constructing an instance of PublicClientApplication.
 
-### Requirements
-* Android 21+
+```javascript
+{
+  "client_id" : "0984a7b6-bc13-4141-8b0d-8f767e136bb7",
+  "redirect_uri" : "msauth://<YOUR_PACKAGE_NAME>/<YOUR_BASE64_URL_ENCODED_PACKAGE_SIGNATURE>",
+  "broker_redirect_uri_registered": true, 
+  //Above is the broker redirect URI format.  If you're upgrading an existing app to the latest MSAL and you are not using the format above.  Set this to false.
```

Comment:
I pushed a code that would detect non-broker Redirect URI and force set this flag to false, so they don't have to do this extra work.

Also broker_redirect_uri_registered is now default to true.